tsl/test/t/006_job_crash_log.pl

```diff
@@ -0,0 +1,65 @@
+# This file and its contents are licensed under the Timescale License.
+# Please see the included NOTICE for copyright information and
+# LICENSE-TIMESCALE for a copy of the license.
+
+use strict;
```

Comment:
Please add a comment on what this test is expected to test. It will make maintaining it in the future easier.

Reply:
Looks good. Resolving comment.